common/changes/@boostercloud/framework-core/read_models_projections_by_readmodel_query_2024-04-19-15-13.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@boostercloud/framework-core",
+      "comment": "Read Models projections by ReadModel query",
+      "type": "minor"
```

Comment:
I'm not sure if this should be tagged as 'minor', since it is a change that extends the possibilities of Read Models with a new feature. Maybe there is a higher type for these changes that fits better? If not, I would say it is OK leaving it this way.

Reply:
If I'm not mistaken, it's minor because it's backward-compatible, whereas a major would imply breaking changes. Read model search in existing code will still work even with this new feature. :wink:

Reply:
OK, makes sense